unity-environment/Assets/ML-Agents/Examples/SharedAssets/Scripts/AdjustTrainingTimescale.cs

```diff
@@ -0,0 +1,58 @@
+﻿//This script lets you change time scale during training. It is not a required script for this demo to function
```

Comment:
Should this be a documented feature ? Is it needed ?

Reply:
I think it is fine to keep as a helper component.